script/lib/env_util.py

```diff
@@ -58,6 +58,29 @@ def get_environment_from_batch_command(env_cmd, initial=None):
   proc.communicate()
   return result
 
+def get_vs_location(vs_version):
+    """
+    Returns the location of the VS building environment.
+
+    The vs_version can be strings like "[15.0,16.0)", meaning 2017, but not the next version.
+    The arch has to be one of "x86", "amd64", "arm", "x86_amd64", "x86_arm", "amd64_x86",
+    "amd64_arm", e.g. the args passed to vcvarsall.bat.
```

Comment:
Nit: "e.g." means "for example". You probably meant to use "i.e."

Reply:
The docs for `vs_get_env` should contain this fix too.

Might also be good to remove duplicate comments by having one function's documentation refer back to the other one for a description? This is a style choice, not a blocker